.drone.yml

```diff
@@ -7071,8 +6785,7297 @@ steps:
+  - docker push "public.ecr.aws/gravitational/teleport:$(echo $DRONE_TAG | sed 's/v//'
+    | cut -d'.' -f 1)-amd64"
+  - docker push "public.ecr.aws/gravitational/teleport:$(echo $DRONE_TAG | sed 's/v//'
+    | cut -d'.' -f 1,2)-amd64"
+  - docker push "public.ecr.aws/gravitational/teleport:$(echo $DRONE_TAG | sed 's/v//'
+    | cut -d'.' -f 1,2,3)-amd64"
+  - docker logout "public.ecr.aws"
+  environment:
+    AWS_ACCESS_KEY_ID:
+      from_secret: PRODUCTION_TELEPORT_DRONE_USER_ECR_KEY
+    AWS_SECRET_ACCESS_KEY:
+      from_secret: PRODUCTION_TELEPORT_DRONE_USER_ECR_SECRET
+  volumes:
+  - name: dockersock
+    path: /var/run
+  depends_on:
+  - Build teleport image "teleport:v10-amd64"
+- name: Tag and push image "teleport:v10-arm" to ECR - production
+  image: docker
+  commands:
+  - apk add --no-cache aws-cli
+  - TIMESTAMP=$(date -d @"$DRONE_BUILD_CREATED" '+%Y%m%d%H%M')
+  - aws ecr-public get-login-password --region=us-east-1 | docker login -u="AWS" --password-stdin
+    public.ecr.aws
+  - docker pull "drone-docker-registry:5000/teleport:$DRONE_TAG-arm"
+  - docker tag "drone-docker-registry:5000/teleport:$DRONE_TAG-arm" "public.ecr.aws/gravitational/teleport:$(echo
+    $DRONE_TAG | sed 's/v//' | cut -d'.' -f 1)-arm"
+  - docker tag "drone-docker-registry:5000/teleport:$DRONE_TAG-arm" "public.ecr.aws/gravitational/teleport:$(echo
+    $DRONE_TAG | sed 's/v//' | cut -d'.' -f 1,2)-arm"
+  - docker tag "drone-docker-registry:5000/teleport:$DRONE_TAG-arm" "public.ecr.aws/gravitational/teleport:$(echo
+    $DRONE_TAG | sed 's/v//' | cut -d'.' -f 1,2,3)-arm"
+  - docker push "public.ecr.aws/gravitational/teleport:$(echo $DRONE_TAG | sed 's/v//'
+    | cut -d'.' -f 1)-arm"
+  - docker push "public.ecr.aws/gravitational/teleport:$(echo $DRONE_TAG | sed 's/v//'
+    | cut -d'.' -f 1,2)-arm"
+  - docker push "public.ecr.aws/gravitational/teleport:$(echo $DRONE_TAG | sed 's/v//'
+    | cut -d'.' -f 1,2,3)-arm"
+  - docker logout "public.ecr.aws"
+  environment:
+    AWS_ACCESS_KEY_ID:
+      from_secret: PRODUCTION_TELEPORT_DRONE_USER_ECR_KEY
+    AWS_SECRET_ACCESS_KEY:
+      from_secret: PRODUCTION_TELEPORT_DRONE_USER_ECR_SECRET
+  volumes:
+  - name: dockersock
+    path: /var/run
+  depends_on:
+  - Build teleport image "teleport:v10-arm"
+- name: Tag and push image "teleport:v10-arm64" to ECR - production
+  image: docker
+  commands:
+  - apk add --no-cache aws-cli
+  - TIMESTAMP=$(date -d @"$DRONE_BUILD_CREATED" '+%Y%m%d%H%M')
+  - aws ecr-public get-login-password --region=us-east-1 | docker login -u="AWS" --password-stdin
+    public.ecr.aws
+  - docker pull "drone-docker-registry:5000/teleport:$DRONE_TAG-arm64"
+  - docker tag "drone-docker-registry:5000/teleport:$DRONE_TAG-arm64" "public.ecr.aws/gravitational/teleport:$(echo
+    $DRONE_TAG | sed 's/v//' | cut -d'.' -f 1)-arm64"
+  - docker tag "drone-docker-registry:5000/teleport:$DRONE_TAG-arm64" "public.ecr.aws/gravitational/teleport:$(echo
```

Comment:
Can we set "major", "major.minor" and "major.minor.patch" versions to some environment variables or write them to files to avoid having to do `sed | cut` magic everywhere?

Reply:
Fixed in [186a98c7452c185030084a93ada7d89757deafaf](https://github.com/gravitational/teleport/pull/16688/commits/186a98c7452c185030084a93ada7d89757deafaf)